autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -1832,13 +1833,20 @@ mod tests {
         };
 
         // Ensure the initial expiry (timestamp) of all the cells in the row
-        let expiry = row.take_required_cell("connected_at").unwrap().timestamp;
+        let connected_at = SystemTime::UNIX_EPOCH
```

Comment:
Sigh. Yeah, this was using cell timestamp instead of the actual value. No idea how this was passing originally.

Reply:
But that's the point of the test: ensuring ttls are updated. The `connected_at` value isn't important here, I only chose it because it's the first field read/write in the serialization methods

Reply:
Ah, if that's the case, then we should make sure that all records, including the `chid:*` items are updated whenever we call `increment_storage`, since that's currently failing. Thanks!